web/src/tippyjs.js

```diff
@@ -309,6 +309,7 @@ export function initialize() {
 
     delegate("body", {
         target: ".narrow_to_compose_recipients",
+        delay: [300, 0],
```

Comment:
Should this be `LONG_HOVER_DELAY`? I think we should have an explanatory comment for any delays that aren't that -- it's good to have a small set of patterns, and an explanation for any variations from that.

Reply:
Yeah.